ixc_whitenoise/middleware.py

```diff
@@ -5,10 +5,17 @@
 from django.http import HttpResponseRedirect
 from django.utils.functional import empty
 from django.utils.six.moves.urllib.parse import urlparse
+
+try:
+    from django.apps import apps
+    get_model = apps.get_model
+except ImportError:
+    # for django prior to 1.7
+    from django.db.models import get_model
+
 from whitenoise.middleware import WhiteNoiseMiddleware
 from whitenoise.utils import ensure_leading_trailing_slash
 
-from ixc_whitenoise.models import UniqueFile
```

Comment:
@alaasalman Can we do `UniqueFile = get_model(...)` up here at the top of the module, instead of inside class methods? Otherwise we might as well just import the actual model inside the class methods to work-around the import error, and forget about using `get_model` entirely?

Reply:
Depends on what coding style you're going for. I can implement your suggestion if that's your preference.